### ISSUE TITLE
sh: -A: command not found

### ISSUE THREAD

**Issue by disrvptor** [oldpatricka/vagrant-xhyve#5]
Whenever I try to up a machine I get the error "sh: -A: command not found". I did some debugging and it looks like the binary is never inserted into build_command. I also noticed the version of xhyve-ruby in `vendor` is different than the official `0.0.6` version. Running on OS X 10.11.6.

```shell
macbook:xhyve-vagrant disrvptor$ vagrant -v
Vagrant 1.8.6
macbook:xhyve-vagrant disrvptor$ vagrant plugin install vagrant-xhyve
Installing the 'vagrant-xhyve' plugin. This can take a few minutes...
Installed the plugin 'vagrant-xhyve (0.3.0)'!
macbook:xhyve-vagrant disrvptor$ sudo vagrant up
Password:
Bringing machine 'default' up with 'xhyve' provider...
==> default: Checking if box 'oldpatricka/ubuntu-14.04' is up to date...
==> default: Importing box...
==> default: Done importing box.
==> default:  About to launch vm...
sh: -A: command not found
Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
macbook:xhyve-vagrant disrvptor$ sudo vagrant up --provider=xhyve
Bringing machine 'default' up with 'xhyve' provider...
==> default: Checking if box 'oldpatricka/ubuntu-14.04' is up to date...
==> default:  About to launch vm...
sh: -A: command not found
macbook:xhyve-vagrant disrvptor$ 
```

Here is my Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "oldpatricka/ubuntu-14.04"

  config.vm.provider :xhyve do |xhyve|
	  xhyve.cpus = 2
	  xhyve.memory = "1G"
	  xhyve.xhyve_binary = "/Applications/Docker.app/Contents/MacOS/com.docker.hyperkit"
  end
end
```

**Comment by disrvptor** [comment]
I upgraded to Vagrant 1.9.0, rebuilt the plugins and still received the same error. I see the embedded version of xhyve-ruby is what's in master at https://github.com/dalehamel/xhyve-ruby, so I copied `guest.rb` from the embedded version into the `xhyve-ruby-0.0.6` directory and things seem to work now.

Is there a way to pull in the dependencies of xhyve-ruby, but not have it as a dependency and use the embedded version?
